tests/house_tests.py

```diff
@@ -88,14 +88,16 @@ def test_profession_either_parent(self):
             city=City.objects.get(name='Tel Aviv'),
             income=5000,
             children=1,
-            parent_profession_2=Job.TEACHER,
+            parent_profession_1=Job.TEACHER,
+            parent_profession_2=Job.OTHER,
         ).save()
         HouseFactory(
             country=Country.objects.get(name='Israel'),
             city=City.objects.get(name='Tel Aviv'),
             income=5000,
             children=1,
-            parent_profession_1=Job.TEACHER,
+            parent_profession_1=Job.OTHER,
+            parent_profession_2=Job.TEACHER,
```

Comment:
Why do we have this change here?

Reply:
This is a fix for a test that is flakey because its not mentioned what the other profession is, so once every time it breaks, this fixes it

Reply:
Ok 
it is not relevant to this PR, can you move it to a small new one and I'll merge?

Reply:
Sure

Reply:
https://github.com/redhat-beyond/Xpense/pull/120